### ISSUE TITLE
Turning on/off a filter adds those types of proposals to the top instead of putting them all together

### ISSUE THREAD

**Issue by RaymondDashWu** [solana-labs/governance-ui#316]
Before
![image](https://user-images.githubusercontent.com/33266041/152672922-74413558-27e6-431f-8806-ed750d1e85f6.png)

After adding drafts to filter
![image](https://user-images.githubusercontent.com/33266041/152672930-877aa030-21ba-411a-aa97-35bbcaf6835e.png)

Expected there to be various completed results as those were more recent

**Comment by appsplash99** [comment]
Just wanted to understand the problem better...

Is the list supposed to be arranged in ascending order with all the list items??

whereas, its currrently its just pushing all the draft items in the starting oof the list items...

am i correct??

**Comment by RaymondDashWu** [comment]
Yup that's correct. Ascending order of dates. Current logic pushes all draft items to the top.

**Comment by appsplash99** [comment]
I can take a look at it.
can you assign it to me ??

**Comment by RaymondDashWu** [comment]
I'm not a maintainer so I don't have the power to assign that. I just noticed the bug when using the UI.

**Comment by masayafunakoshi1** [comment]
Actually, it's probably better to keep the drafts at the top of the list. Because the only reason people might want to see the drafts is to access them again to deploy a proposal. So it would make sense it goes to the top.
